python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -214,17 +214,42 @@ def convert_fully_connected(node, **kwargs):
     onnx = import_onnx_modules()
     name = node["name"]
     inputs = node["inputs"]
+    attrs = node["attrs"]
+    initializer = kwargs["initializer"]
+
+    no_bias = 1 if ("no_bias" in attrs) and \
```

Comment:
please functionize all similar calls to improve readability and reduce redundancy

Reply:
there are 3 such checks (including this) in the current code. Will create a common function. Thanks.